bin/travis/prepare_behat.sh

```diff
@@ -7,10 +7,13 @@ set -e
 # Change local git repo to be a full one as we will reuse it for composer install below
 git fetch --unshallow && git checkout -b tmp_ci_branch
 export BRANCH_BUILD_DIR=$TRAVIS_BUILD_DIR TRAVIS_BUILD_DIR="$HOME/build/ezplatform"
+
+# Checkout meta repo, use the branch indicated in composer.json under extra._ezplatform_branch_for_behat_tests
+EZPLATFORM_BRANCH=`php -r '$composer = json_decode(file_get_contents("./composer.json")); echo $composer->extra->_ezplatform_branch_for_behat_tests;'`
```

Comment:
You could shorten that to `php -r 'echo json_decode(file_get_contents("./composer.json"))->extra->_ezplatform_branch_for_behat_tests;'`.